content/_code-samples/checks/py/create_check.py

```diff
@@ -0,0 +1,95 @@
+from xrpl.wallet import Wallet, generate_faucet_wallet
+from xrpl.clients import JsonRpcClient
+from xrpl.models import CheckCreate, IssuedCurrencyAmount
+from xrpl.transaction import (safe_sign_and_autofill_transaction,
+                              send_reliable_submission)
+from xrpl.utils import datetime_to_rippletime, xrp_to_drops
```

Comment:
typo here, 
```suggestion
from xrpl.utils import datetime_to_rippletime, xrp_to_drops
```
```
from xrpl.utils import datetime_to_ripple_time, xrp_to_drops
```
https://xrpl-py.readthedocs.io/en/stable/source/xrpl.utils.html?highlight=datetime#xrpl.utils.datetime_to_ripple_time